api/views/personal_info.py

```diff
@@ -19,8 +21,40 @@ def get_serializer_class(self) -> type[BaseSerializer[PersonalInfo]]:
     @action(detail=False, methods=["post"])
     def check_existence(self, request: Request) -> Response:
         """
-        Check if a personal info exists.
+        Checks if a personal info exists.
         """
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
         return Response(status.HTTP_200_OK)
+
+    @extend_schema(
+        description=(
+            "Checks if personal info exists with this chat ID of Telegram registration bot."
+        ),
+        parameters=[
+            OpenApiParameter(name="chat_id", type=str, required=True),
+        ],
+        responses={
+            200: OpenApiResponse(description="Chat ID is found"),
+            404: OpenApiResponse(description="Chat ID not found"),
+        },
+    )
+    @action(detail=False, methods=["get"])
+    def check_existence_of_chat_id(self, request: Request) -> Response:
+        """
+        Check if PersonalInfo with given ``registration_telegram_bot_chat_id`` exists.
+        """
+        # This method is used without a serializer because this seems to be overkill
```

Comment:
I think this entire comment could be removed, it's too long and not needed in my opinion